src/test/java/com/epam/jamp2/service/impl/CalculationCommandExecutionServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+package com.epam.jamp2.service.impl;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.epam.jamp2.model.CalculationCommand;
+import com.epam.jamp2.model.CommandFormatException;
+import com.epam.jamp2.model.FixerioResponse;
+import com.epam.jamp2.model.Value;
+import com.epam.jamp2.rest.FixerioServiceProxy;
+import com.epam.jamp2.service.CalculationCommandExecutionService;
+import com.epam.jamp2.service.FxRatesService;
+
+import retrofit2.Call;
+import retrofit2.Response;
+
+
+@RunWith(SpringRunner.class)
```

Comment:
Why did you use this runner? Is it worth initializing whole Spring context in some of your tests?